indico/web/client/js/react/components/WTFDurationField.jsx

```diff
@@ -0,0 +1,65 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2021 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import TimePicker from 'rc-time-picker';
+import React, {useEffect, useMemo, useRef, useState, useCallback} from 'react';
+
+const timeToSeconds = time => (time ? time.diff(moment().startOf('day'), 'seconds') : null);
+const secondsToTime = seconds =>
+  moment()
+    .startOf('day')
+    .seconds(seconds);
```

Comment:
Just wondering.. will this work correctly on days with a DST change?

Reply:
Looks like it does, as adding seconds to a DST day works just as with a normal day:

```javascript
moment('2021-03-28 08:00').startOf('day').seconds('43200').toDate()
Sun Mar 28 2021 12:00:00 GMT+0200 (Central European Summer Time)
moment('2021-03-27 08:00').startOf('day').seconds('43200').toDate()
Sat Mar 27 2021 12:00:00 GMT+0100 (Central European Standard Time)
```
When using `diff`, it accounts for the missing hour:

```javascript
a = moment('2021-03-28 00:00')
Moment {_isAMomentObject: true, _i: "2021-03-28 00:00", _f: "YYYY-MM-DD HH:mm", _isUTC: false, _pf: {…}, …}
b = moment('2021-03-28 23:59:59')
Moment {_isAMomentObject: true, _i: "2021-03-28 23:59:59", _f: "YYYY-MM-DD HH:mm:ss", _isUTC: false, _pf: {…}, …}
b.diff(a, 'hour')
22
```
Because `a` and `b` are different timezones:
```javascript
a.toDate()
Sun Mar 28 2021 00:00:00 GMT+0100 (Central European Standard Time)
b.toDate()
Sun Mar 28 2021 23:59:59 GMT+0200 (Central European Summer Time)
```